src/cards/LightCard.ts

```diff
@@ -36,16 +40,14 @@ class LightCard extends AbstractCard {
   /**
    * Class constructor.
    *
-   * @param {hassEntity} entity The hass entity to create a card for.
-   * @param {lightCardOptions} [options={}] Options for the card.
+   * @param {EntityRegistryEntry} entity The hass entity to create a card for.
+   * @param {cards.LightCardOptions} [options={}] Options for the card.
    * @throws {Error} If the Helper module isn't initialized.
    */
-  constructor(entity, options = {}) {
+  constructor(entity: EntityRegistryEntry, options: cards.LightCardOptions = {}) {
```

Comment:
Double Tap action does not work anymore. Before you were using a `try` to assign it  in the [Abstract Card](https://github.com/AalianKhan/mushroom-strategy/blob/ccf10f716c9dc2bb4716a3ef6250aba626fa9c4a/src/cards/AbstractCard.js#L63)

Reply:
I'm pleased to see your testing the code thoroughly.
I'll look into it. It think I forgot about card with actions by mistake.
Can you please verify the actions of the other cards also?
(Vacuum, Switch, Lights are the ones with an action, I believe)